config/quality.gradle

```diff
@@ -9,6 +9,27 @@ check.dependsOn 'checkstyle', 'findbugs', 'pmd', 'lint'
 def configDir = "${project.rootDir}/config/quality"
 def reportsDir = "${project.buildDir}/reports"
 
+
+checkstyle {
```

Comment:
Was just messing around to get rid of PMD/Findbugs errors for the androidTests ended up reordering this file. So now the plugin configurations come before the tasks...